adafruit_gps.py

```diff
@@ -675,8 +683,13 @@ class GPS_GtopI2C(GPS):
     """
 
     def __init__(
-        self, i2c_bus, *, address=_GPSI2C_DEFAULT_ADDRESS, debug=False, timeout=5
-    ):
+        self,
+        i2c_bus: I2C,
+        *,
+        address: int = _GPSI2C_DEFAULT_ADDRESS,
+        debug: bool = False,
+        timeout: float = 5,
```

Comment:
should timeout allow `int` or `float` Or maybe have it's default value changed to `5.0` instead?